locksmith/migrations/20230203140302-custom-email-content.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('CustomEmailContents', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      lockAddress: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      network: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+      },
+      template: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      content: {
+        type: Sequelize.STRING,
```

Comment:
I wonder if this is long enough to support long texts?

Reply:
ah, let me actually change it to `TEXT`